index.html

```diff
@@ -56,25 +62,226 @@ <h1>
+                <a href="#"
+                  ><img src="./images/Linkedin icon.svg" alt="linkedin-logo"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="./images/medium.svg" alt="medium-logo"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="./images/github.svg" alt="github-logo"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="./images/angelist.svg" alt="angelist-logo"
+                /></a>
+              </li>
             </ul>
           </div>
         </div>
-          
-       
       </section>
-     </header>
+
+      <section id="works">
+        <div class="card">
+          <div class="head-image">
+            <img
+              src="./images/Snapshoot Portfolio.png"
+              alt="work-image-project"
+            />
+            <h1 class="tonic">TONIC</h1>
+            <ul class="dates">
+              <li>CANOPI</li>
+              <li><img src="./images/bullet.svg" alt="counter" /></li>
+              <li class="stack-backend">Back End Dev</li>
+              <li><img src="./images/bullet.svg" alt="counter" /></li>
+              <li class="stack-backend">2015</li>
+            </ul>
+
+            <p class="paragraph">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="stacks">
+              <li class="stack-over">HTML</li>
+              <li class="stack-over">CSS</li>
+              <li class="stack-over">Javascript</li>
+            </ul>
+            <a href="#" class="btn">See Project</a>
+          </div>
+        </div>
+        <div class="card">
+          <div class="head-image">
+            <img src="./images/Snapshoot 2.png" alt="work-image-project" />
+            <h1 class="tonic">Multi-Post <br />Stories</h1>
+            <ul class="dates">
+              <li>CANOPI</li>
+              <li><img src="./images/bullet.svg" alt="counter" /></li>
+              <li class="stack-backend">Back End Dev</li>
+              <li><img src="./images/bullet.svg" alt="counter" /></li>
+              <li class="stack-backend">2015</li>
+            </ul>
+
+            <p class="paragraph">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="stacks">
+              <li class="stack-over">HTML</li>
+              <li class="stack-over">CSS</li>
+              <li class="stack-over">Javascript</li>
+            </ul>
+            <a href="#" class="btn">See Project</a>
+          </div>
+        </div>
+        <div class="card">
+          <div class="head-image">
+            <img src="./images/Snapshoot 3.png" alt="work-image-project" />
+            <h1 class="tonic">TONIC</h1>
```

Comment:
- [ ] Just as I suggest above You should use an `h2 or h3` tag instead of an h1. You are meant to have only 1 `h1` on your page and that should be the main title. By using appropriate heading levels, you can help search engines understand the main topics and subtopics of a page, which can improve its ranking in search results. :+1: :+1: